$OpenBSD: patch-Lib_test_regrtest_py,v 1.6 2014/01/09 18:03:35 fgsch Exp $

- test_bsddb and test_bsddb3 are in the list below as they are only available
  on i386 and amd64.

--- Lib/test/regrtest.py.orig	Sun Nov 10 07:36:40 2013
+++ Lib/test/regrtest.py	Fri Nov 22 02:36:23 2013
@@ -1416,22 +1416,16 @@ _expectations = {
         """,
     'openbsd3':
         """
-        test_ascii_formatd
         test_bsddb
         test_bsddb3
         test_ctypes
+        test_curses
         test_dl
         test_epoll
-        test_gdbm
-        test_locale
-        test_normalization
+        test_gdb
         test_ossaudiodev
-        test_pep277
-        test_tcl
         test_tk
         test_ttk_guionly
-        test_ttk_textonly
-        test_multiprocessing
         """,
     'netbsd3':
         """
@@ -1458,6 +1452,8 @@ _expectations['freebsd5'] = _expectations['freebsd4']
 _expectations['freebsd6'] = _expectations['freebsd4']
 _expectations['freebsd7'] = _expectations['freebsd4']
 _expectations['freebsd8'] = _expectations['freebsd4']
+_expectations['openbsd4'] = _expectations['openbsd3']
+_expectations['openbsd5'] = _expectations['openbsd3']
 
 class _ExpectedSkips:
     def __init__(self):
